dev-libs/bglibs/bglibs-2.04-r3.ebuild

```diff
@@ -30,6 +30,8 @@ PATCHES=(
 	"${FILESDIR}"/bglibs-2.04-stack-buffers.patch
 	"${FILESDIR}"/bglibs-2.04-feature-tests.patch
 	"${FILESDIR}"/bglibs-2.04-musl.patch
+	"${FILESDIR}"/bglibs-2.04-gcc14.patch
+	"${FILESDIR}"/bglibs-2.04-prefix.patch
```

Comment:
Please revbump by default for these Modern C fixes, per https://wiki.gentoo.org/wiki/Modern_C_porting#Is_this_cosmetic.3F.

Reply:
Done